src/components/measureNumberFormat/syntaxHighlighting/SyntaxHighlighting.tsx

```diff
@@ -0,0 +1,82 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { UnControlled as CodeMirrorInput } from "react-codemirror2";
+import * as CodeMirror from "codemirror";
+import "codemirror/lib/codemirror.css";
+import "codemirror/addon/mode/simple";
+import "codemirror/addon/edit/matchbrackets";
+
+const measureFormatMode = (CodeMirror as any).defineSimpleMode("numberFormat", {
```

Comment:
The comments seems to be from an example and are not related to our implementation so I would remove them
see https://github.com/google/comid/blob/master/example/mode/simplemode.html